### PR TITLE
[USH-3425] Updated user-controlled repeat group buttons in web apps

### PR DESCRIPTION
**Pull request by orangejenny**
## Product Description
A couple of UX tweaks to user-controlled repeat groups in Web Apps:
* Increase buttons to standard size
* Replace delete text with an X icon
* Add a plus icon to the add button
* Add a light border between repeat groups

https://dimagi-dev.atlassian.net/browse/USH-3425

### Safety story
This is web apps UI, but it's largely HTML. The main risk would have been a syntax error. Locally tested.

### Automated test coverage

Unlikely

### QA Plan

Not requesting QA

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/cloudcare/templates/form_entry/templates.html [hunk 1]
```diff
@@ -310,9 +310,12 @@ <h3 class="caption" data-bind="html: header" tabindex="0"></h3>
...
     </div>
     <div class="panel-footer">
-      <button class="btn btn-default btn-xs add" href="#"
-              data-bind="click: newRepeat, text: getTranslation('repeat.dialog.add.new', '{% trans_html_attr 'Add new repeat' %}')"
-              id="repeat-add-new" />
+      <button class="btn btn-default add" href="#"
+              data-bind="click: newRepeat"
+              id="repeat-add-new">
+              <i class="fa fa-plus"></i>
+              <!-- ko text: getTranslation('repeat.dialog.add.new', '{% trans_html_attr 'Add new repeat' %}') --><!-- /ko -->
```
Comment (MartinRiese) [inline corehq/apps/cloudcare/templates/form_entry/templates.html:317]: Could this comment be removed?

**Review by MartinRiese — APPROVED** [review]

**Review by nospame — APPROVED** [review]

**Comment by orangejenny** [comment]
Added a commit to put borders between repeats:
Screen Shot 2023-07-18 at 1 18 07 PM

File: corehq/apps/hqwebapp/static/cloudcare/less/formplayer-common/form.less [hunk 2]
```diff
@@ -38,3 +38,8 @@ legend {
     display: none;
   }
 }
+
+.gr.repetition:not(:first-child) {
```
Comment (MartinRiese) [inline corehq/apps/hqwebapp/static/cloudcare/less/formplayer-common/form.less:42]: `not(:first-child)`: That's pretty sweet

File: corehq/apps/cloudcare/templates/form_entry/templates.html (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/cloudcare/templates/form_entry/templates.html:317]: That's knockout "containerless syntax": https://knockoutjs.com/documentation/text-binding.html

It performs slightly better than creating a span that just has a text binding.

**Pull request merged**